.storybook/stories/shaderMaterial.stories.ts

```diff
@@ -7,71 +7,68 @@ import { OrbitControls } from 'three-stdlib'
 export default {
   title: 'Shaders/shaderMaterial',
   argTypes: { repeats: { control: { type: 'range', min: 1, max: 5, step: 1 } } },
-} as Meta
+} as Meta // TODO: this should be `satisfies Meta` but commit hooks lag behind TS
```

Comment:
This requires TypeScript 4.9, but may need bumping of prettier/eslint. Commit hooks were hanging my commits so I conceded correctness here.